### PR TITLE
Added apigee endpoints

### PR DESCRIPTION
**Pull request by svidhani**
Added endpoints for Apigee integration with JAZZ. This will allow JAZZ to point to Apigee if user has made that selection

### CODE REVIEW

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json [hunk 1]
```diff
@@ -93,5 +93,17 @@
 			 "logs": false,
 			 "assets": false
 		 }
-   }
+   },
+    "APIGEE": {
+        "endpoints": {
```
Comment (devsatishm) [inline installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: Let's have these configurations structured similar to how we have for AWS API Gateway. 
"APIGEE": {
  	"API_ENDPOINTS": {
    		"PROD": {APIGEE_PROD_API_ENDPOINT} ......

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json [hunk 2]
```diff
@@ -93,5 +93,17 @@
...
+            "prod": "{APIGEE_PROD_API}",
+            "test": "{APIGEE_TEST_API}",
+            "stg": "{APIGEE_STG_API}",
+            "dev01": "{APIGEE_DEV01_API}",
```
Comment (devsatishm) [inline installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: We'll only have PROD, STG & DEV. Others will not be used by Jazz.

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json [hunk 3]
```diff
@@ -93,5 +93,17 @@
 			 "logs": false,
 			 "assets": false
 		 }
-   }
+   },
+    "APIGEE": {
+        "endpoints": {
+            "prod": "{APIGEE_PROD_API}",
```
Comment (devsatishm) [inline installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: CAPS for keys?

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json [hunk 4]
```diff
@@ -93,5 +93,17 @@
 			 "logs": false,
 			 "assets": false
 		 }
-   }
+   },
+    "APIGEE": {
```
[the remark this answers is not in the document]
Comment (svidhani) [reply installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: yes i know and adding what we have for now

File: installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json (hunk 2 above)
Comment (svidhani) [reply installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json]: why not test, we can use it for testing(unit/integration)

**Pull request merged**